package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "dropbox-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "tsc && node ./dist/index.js",
+    "serve": "./dist/index.js",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/axios": "^0.14.0",
+    "@types/elastic-apm-node": "^2.7.0",
+    "@types/lodash": "^4.14.165",
+    "@types/mongoose": "^5.10.2",
+    "@types/node": "^14.14.10",
+    "grpc-ts-health-check": "^1.0.14",
+    "tslint": "^6.1.3",
```

Comment:
Just so you know, [TSLint has been deprecated since 2019](https://palantir.github.io/tslint/). 😃

Reply:
Also, you don't have a `tslint` command anywhere in your scripts, and even if you would have, it would probably fail 🙊

Reply:
removed.

Reply:
But know you don't lint at all! 🙀